gatorgrade/main.py

```diff
@@ -27,10 +32,12 @@ def generate(
         "--force",
         "-f",
         help="Force gatorgrade to overwrite an existing yml file.",
+        type = [" "],
     )
 ):
     """Generate a gatorgrade.yml file."""
     if force is False:
+        checks = generate_config(target_path_list, relative_run_path)
```

Comment:
The values `target_path_list` and `relative_run_path` are not defined here. `target_path_list` should be a list of strings captured through a command-line argument and `relative_run_path` does not need to be specified--we can just use its default value.
Additionally, the return value of `generate_config` probably does not need to be captured.